plover/messages/es/LC_MESSAGES/plover.po

```diff
@@ -2,191 +2,262 @@ msgid ""
+"Developers:\n"
+"\n"
+"Joshua Lifton\n"
+"Hesky Fisher\n"
+"Ted Morin\n"
+"Benoit Pierre\n"
+"\n"
+"and many more on GitHub:\n"
+"<https://github.com/openstenoproject/plover>"
+msgstr ""
+"Fundado por Mirabai Knight, estenotipista.\n"
+"\n"
+"Desarrolladores:\n"
+"\n"
+"Joshua Lifton\n"
+"Hesky Fisher\n"
+"Ted Morin\n"
+"Benoit Pierre\n"
+"\n"
+"Y muchos más en GitHub:\n"
+"<https://github.com/openstenoproject/plover>"
+
+#. Short description for Plover, currently not used in the interface.
+#: plover/__init__.py:33
+msgid "Open Source Stenography Software"
+msgstr "Software de estenotipia de código abierto"
+
+#: plover/__init__.py:34
+msgid ""
+"Plover is a free open source program intended to bring realtime\n"
+"stenographic technology not just to stenographers, but also to\n"
+"hackers, hobbyists, accessibility mavens, and all-around speed demons."
+msgstr ""
+"Plover es un programa libre de código abierto que tiene el objetivo de "
+"llevar la tecnología de estenotipia en tiempo real,\n"
+"no solo a estenotipistas, sino también\n"
+"a hackers, personas aficionadas, expertas en accesibilidad y genios de la "
+"velocidad por todas partes."
+
+#. Widget: “AboutDialog”, windowtitle.
+#: plover/gui_qt/about_dialog_ui.py:38
 msgid "Plover: About"
 msgstr "Plover: Acerca de"
 
-#: plover/gui_qt/add_translation_dialog.py:13
+#. Widget: “AddTranslationDialog”, tooltip.
+#. Widget: “AddTranslationWidget”, tooltip.
+#: plover/gui_qt/add_translation_dialog.py:8
+#: plover/gui_qt/add_translation_widget.py:22
+msgid "Add a new translation to the dictionary."
+msgstr "Añadir una nueva traducción al diccionario."
+
+#: plover/gui_qt/add_translation_dialog.py:10
 msgid "Add Translation"
 msgstr "Añadir traducción"
 
-#: plover/gui_qt/add_translation_dialog_ui.py:40
-msgid "Plover: Add Translation"
-msgstr "Plover: Añadir traducción"
-
-#: plover/gui_qt/add_translation_widget.py:200
-msgid "disabled"
-msgstr "deshabilitado"
+#. Widget: “AddTranslationWidget”.
+#: plover/gui_qt/add_translation_widget.py:199
+msgid "{dictionary} (disabled)"
+msgstr "{dictionary} (desactivado)"
 
+#. Widget: “AddTranslationWidget”.
 #: plover/gui_qt/add_translation_widget.py:258
 msgid "{strokes} maps to "
 msgstr "{strokes} se asigna a "
 
-#: plover/gui_qt/add_translation_widget.py:268
-msgid "<br />Overwritten entries:"
-msgstr "<br /> Entradas sobrescritas:"
+#. Widget: “AddTranslationWidget”.
+#: plover/gui_qt/add_translation_widget.py:269
+msgid "Overwritten entries:"
+msgstr "Entradas sobrescritas:"
 
-#: plover/gui_qt/add_translation_widget.py:272
+#. Widget: “AddTranslationWidget”.
+#: plover/gui_qt/add_translation_widget.py:274
 msgid "{strokes} is not mapped in any dictionary"
 msgstr "{strokes} no está asignada en ningún diccionario"
 
-#: plover/gui_qt/add_translation_widget.py:287
+#. Widget: “AddTranslationWidget”.
+#: plover/gui_qt/add_translation_widget.py:290
 msgid "{translation} is mapped to: {strokes}"
 msgstr "\"{translation}\" está asignada a {strokes}"
```

Comment:
Are the extra `"` necessary? The one right below does not have them.

Reply:
> 
> 
> Are the extra `"` necessary? The one right below does not have them.


No, it was a mistake from my part. You can request changes (adding a suggestion with control+g) so I can accept the suggestion from GitHub or I'll fix the file and update it.
Thanks

Reply:
```suggestion
msgstr "{translation} está asignada a {strokes}"
```